integration-tests/simnode/src/tests/runtime_upgrade.rs

```diff
@@ -52,7 +52,7 @@ where
 	println!("\nold_runtime_version: {}\n", old_runtime_version);
 
 	let call = sudo::Call::sudo_unchecked_weight {
-		call: Box::new(system::Call::set_code { code }.into()),
+		call: Box::new(system::Call::set_code_without_checks { code }.into()),
```

Comment:
why without checks?

Reply:
Former Dali runtime had spec_name of Picasso. Changing it to Dali requires without checks